optimum/exporters/tasks.py

```diff
@@ -1552,6 +1553,18 @@ def infer_task_from_model(
 
         return task
 
+    @staticmethod
+    def _infer_library_from_model(model: Union["PreTrainedModel", "TFPreTrainedModel"]):
```

Comment:
was thinking about renaming `infer_library_from_model` to `infer_library_from_model_name_or_path` but don't want to introduce braking changes, an other solution would be to follow same pattern as https://github.com/huggingface/optimum/blob/36790792f65cbf9377c34c65617694c67006c105/optimum/exporters/tasks.py#L1519